[Worker] cron triggers revision (#2637)
cloudflare/cloudflare-docs: products/workers/src/content/platform/cron-triggers/index.md

## Background

Cron Triggers allow users to map a cron expression to a Worker script using a [ScheduledEvent](/runtime-apis/scheduled-event) listener that enables Workers to be executed on a schedule. Cron Triggers are ideal for running periodic jobs for maintenance or calling third-party APIs to collect up-to-date data. Workers scheduled by Cron Triggers will run on underutilized machines to make the best use of Cloudflare's capacity and route traffic efficiently.

## Adding Cron Triggers

> Cron Triggers can be added to scripts with the Cloudflare API, or in the dashboard in **Workers** > **Manage Workers** > select your Worker > [**Triggers**](https://dash.cloudflare.com/?to=/:account/workers). Refer to [Limits](/limits#number-of-schedules) to track the maximum number of Cron Triggers per Worker. If a script is managed with Wrangler, Cron Triggers should be exclusively managed through the `wrangler.toml` file.

Reviewer: ```suggestion
You can add Cron Triggers to scripts with the Cloudflare API, or in the dashboard in **Workers** > **Manage Workers** > select your Worker > [**Triggers**](https://dash.cloudflare.com/?to=/:account/workers). Refer to [Limits](/limits#number-of-schedules) to track the maximum number of Cron Triggers per Worker. If a script is managed with Wrangler, Cron Triggers should be exclusively managed through the `wrangler.toml` file.
```

<Aside type="note" header="Requires a ScheduledEvent Listener">

> To respond to a Cron Trigger, a [`"scheduled"` event](/runtime-apis/scheduled-event) listener must be added to the Workers script.

Reviewer: ```suggestion
To respond to a Cron Trigger, you must add a [`"scheduled"` event](/runtime-apis/scheduled-event) listener to the Workers script.
```

</Aside>

![workers-schedule-editor](./media/workers-schedule-editor.png)

## Supported cron expressions

> Cloudflare support cron expressions with 5 fields, along with most [Quartz scheduler](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html#introduction)-like cron syntax extensions:

Reviewer: ```suggestion
Cloudflare supports cron expressions with five fields, along with most [Quartz scheduler](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html#introduction)-like cron syntax extensions:
```
